### PR TITLE
grc: fix dark theme on ParamWidgets

### LINKED ISSUE #2374: GRC: Dark theme not working well, input box text is white
I use dark gtk theme and for me I can't see text in white input boxes as the text becomes white as well. I can see it in the inputboxes that are colored. Is there somewhere that text color is hardcoded?

image

I selected the rest of the test in the "Extra Dict." field, such that you can see that there are text input.

3.8tech-preview-280-g3a533f78

### PR DESCRIPTION
**Pull request by chuangzhu**
Fix `gtk-application-prefer-dark-theme` not affecting the color displayed.

For example, when you have a `~/.config/gtk-3.0/settings.ini` like this, the background color of the input box will still be white.

```
gtk-application-prefer-dark-theme=true
gtk-theme-name=Breeze
```

This also fixes #2374.

### CODE REVIEW

**Comment by mormj** [comment]
@chuangzhu - thank you for the contribution.  GNU Radio uses the DCO as described here: https://github.com/gnuradio/gnuradio/blob/master/CONTRIBUTING.md which basically requires that you sign your commit using `git commit -s`

This can be done easily with `git commit -s --amend` on this branch and then force pushing.

Thanks!

**Comment by mormj** [comment]
The logic makes sense here in that if you have the gtk setting `gtk-application-prefer-dark-theme=true` then it should default to a dark them, so that
- If a user has selected a dark theme, this method will return True
- If a user has config option "prefer dark theme" then this will always return true

I don't have a good setup here with dark themes and haven't reproduced the original issue - going to get an additional set of eyes on the logic here.

File: grc/gui/ParamWidgets.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def is_dark_theme(theme_name):
     if prefer_dark in ('1', 'yes', 'true', 'on'):
         theme_name = config.get(
```
Comment (mormj) [inline grc/gui/ParamWidgets.py:33]: If this if statement always returns true, then finding the theme name should not be necessary - this doesn't seem right as this theme name is not used anywhere else in the code.

**Review by mormj — APPROVED** [review]
This logic makes sense now

**Review by willcode — APPROVED** [review]
This works.

If we want to, we can replace the subprocess part, then move that check earlier to make better logic, using the programmatic interface:
```
settings = gtk.Settings.get_default()
theme = settings.get_property('gtk-theme-name')
```
There ought to be a way to check the "prefer" value to, but I didn't find it right away.

**Pull request merged**